packages/compass-components/README.md

```diff
@@ -1,17 +1,13 @@
-# @mongodb-js/compass-components [![][npm_img]][npm_url]
+# @mongodb-js/compass-components
 
-> A set of React Components which help build Compass
+Compass Components package contains all foundational components and hooks used to build parts of Compass.
```

Comment:
Would love it if we could write down the reason why we have a single package for all our building block components – if it's pure convenience, that's still a good reason, but if we write it down, we know not to be scared of changing it in the future if something else becomes more convenient 🙂

Reply:
Good point! Something like this?

```suggestion
Compass Components package contains all foundational components and hooks used to build parts of Compass. Keeping all leafygreen dependencies and core UI components in a single package allows us to easier manage external depedendencies on LeafyGreen (including making sure that we never have multiple versions of the same LeafyGreen packages in the application at the same time as this can lead to technical issues) and make sure that the UI is consistently updated throughout the whole application.
```